Mezzurite.React/src/with-mezzurite.hoc.js

```diff
@@ -84,14 +84,15 @@ const withMezzurite = (WrappedComponent) => {
         componentDidMount(){
             let el = ReactDOM.findDOMNode(this.wrappedRef)
             this.fullName = MezzuriteReactUtils.getName(this.displayName, this.key);
+            window.mezzurite.elementLookup[this.fullName] = el;

```

Comment:
This is what you explained earlier, right? The caching of elements to access outside of the HOC?